flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogDataTypeTest.java

```diff
@@ -124,8 +122,8 @@ public void testDataTypes() throws Exception {
 	@Test
 	public void testNonExactlyMatchedDataTypes() throws Exception {
```

Comment:
Maybe we should change this test to a negative test case, as BINARY and VARBINARY are not supported by hive and it's useful to capture the limitation with a test case.